### DISCUSSION TITLE
Help needed! The repo of a popular web framework is unaccessible due to inadvertent account suspension

### DISCUSSION THREAD

**Discussion by helenliangzhu** [community/community#114684] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Dear GitHub support,

I'm the creator (user name: @Tao-VanJS) of a popular web framework - VanJS, which has a solid user base around the world. Starting from March 27 2024, my GitHub account was suspended. I received an email with title "Password reset notification due to suspicious activity" at the same time. All the GitHub repositories I created became unaccessible. The website https://vanjs.org/, which was initially hosted by GitHub pages, was also down because of that. It was recovered after I did an emergency migration to Deno Deploy.

I filed the support ticket to reinstate my account and I really appreciate if GitHub team can prioritize the ticket. Many users rely on VanJS on day-to-day basis. The library has gained 3.4k stars less than a year and is serving 7 million requests / month via CDN. Unable to access the GitHub repository does not only impact the development and maintenance of the project, it also affect its users as they are not able to access the source code, raise issues, or reach out for help when needed. To make things worse, even my personal account, which was used for Deno Deploy, was suspended as well. So I am not even able to edit the https://vanjs.org/ website to broadcast any updates about the situation.

There are users of VanJS sending emails to me asking what happened to the project. I understand their trouble but with all GitHub accounts suspended, I can do very little to help. Without my account reinstated there will be irreversible harm to the users and the community. I don't think I have violated any policies while using the GitHub accounts. So I really appreciate a expedited review and reinstatement of my account.

To add some more context if it helps, here is an introduction of the framework from the Internet: https://thenewstack.io/vanjs-a-minimalist-vanilla-javascript-ui-framework/. Here is a 7-minute video for it: https://www.youtube.com/watch?v=Oh2IEVqarHs.

Really appreciate your help!

**Comment by takien** [comment]
I'm very sad to hear that, I hope the problem is resolved quickly.
I just learned Vanjs.

**Comment by eaallen** [comment]
I love VanJs. I was actually writing a library very similar to it when someone introduced vanJs to me. It is awesome! Github team, please get this issue resolved.
